src/java/Controller/SendEmail.java

```diff
@@ -57,14 +58,21 @@ protected PasswordAuthentication getPasswordAuthentication() {
             MimeMessage message = new MimeMessage(session);
             message.setFrom(new InternetAddress(user));
             message.addRecipient(Message.RecipientType.TO, new InternetAddress(to));
-            message.setSubject("Employee Performance");
-
+            String cc = getRM(email);
+            if(!cc.equals(""))
+                message.addRecipient(Message.RecipientType.CC, new InternetAddress(cc));
+            
+            String[] startEndDates = getStartEnd(email);
+            message.setSubject("Performica Report["+ getStreamName(email) + "]["+ startEndDates[0] + "]-["+startEndDates[1]+"]");
 
+            
             // Create the message part
             BodyPart messageBodyPart = new MimeBodyPart();
 
             // Now set the actual message
-            messageBodyPart.setText("Attached to this email is the pdf doc of the employee's performance.");
+            messageBodyPart.setText("Congratulations, " + getName(email) + "\nHere is your Performica Report! Inside is the score of each module you completed and a small graph"+
+            " to show where you stand compared to the class average. We hope you enjoyed your experience in our " +
+            " training program and wish you the best for your future!");

```

Comment:
I feel like there should be some sort of signature and salutation, but this is good enough for now.